http_server/server_http.py

```
@@ -0,0 +1,34 @@
+import socket
+from http_class import HTTP
+from http_request_class import Request
+http = HTTP()
+request_handler = Request(".\\root", "index.html")
+
+IP = "192.168.1.154"
+port = 80
+
+
+def handle_request(client_sock):
+    if http.get_check_request(client_sock):
+        if http.request[0] == "GET":
+            request_handler.take_request(http.request)
+            request_handler.check_get_send(client_sock)
+        elif http.request[0] == "POST":
```

no handeling post? why is that?

---

maybe return a error and not just pass?